image_proc/src/image_proc.cpp

```diff
@@ -75,33 +54,21 @@ int main(int argc, char * argv[])
 
   rclcpp::init(argc, argv);
 
-  std::string camera_namespace;
-  if (rcutils_cli_option_exist(argv, argv + argc, "--camera_namespace")) {
```

Comment:
Why did we remove this available to remap on commandline?

Reply:
I thought the standard was using ROS parameters rather than command-line options?

Reply:
That was my thinking as well, to normalize it to work like other ROS2-packages and use the `__ns:=/namespace` argument.

Reply:
@SteveMacenski I fully understand we don't want to change too many parameters/arguments between the different version to make it easier for ppl. However before I go about changing stuff for this and put it additional broken up PRs, I'd just like to know the reason for having "camera_namespace parameter, and not using the `__ns:=` functionality instead ?